cusub_perception/localizer/src/localizer.cpp

```diff
@@ -106,6 +130,19 @@ namespace localizer_ns
       print_str = print_str + std::string("\033[93m[WARN] ") + str + std::string("\033[0m");
       ROS_INFO( print_str.c_str());
     }
+    void Localizer::set_not_nodelet(void)
+    {
+      is_nodelet = false;
+    }
 }
 
 PLUGINLIB_EXPORT_CLASS(localizer_ns::Localizer, nodelet::Nodelet);
+
+int main(int argc, char **argv)
+{
+  ros::init(argc, argv, "localizer_node");
+  localizer_ns::Localizer l;
+  l.set_not_nodelet();
```

Comment:
Surely there is less stupid way to check this?

Reply:
maybe, it's not great but it does get the job done